Changelog.md

```diff
@@ -1,5 +1,10 @@
 ## Ignition Rendering
 
+### Ignition Rendering 6.6.3 (2024-01-19)
+
+1. Revert mesh viewer background color back to gray
```

Comment:
Ouch, somehow the source_changelog script ran was pretty broken or I did something badly:
```
gz-rendering on  ign-rendering6 via △ v3.22.1 ❯ ~/code/release-tools/source-repo-scripts/source_changelog.bash 6.6.2
Changes since 6.6.2
1. Prepare release 6.6.3
    * [Pull request #958](https://github.com/gazebosim/gz-rendering/pull/958)

1. Backport ogre2 lidar performance improvement
    * [Pull request #956](https://github.com/gazebosim/gz-rendering/pull/956)

1. Prepare for 3.7.2 release
    * [Pull request #952](https://github.com/gazebosim/gz-rendering/pull/952)

1. Update github action workflows
    * [Pull request #940](https://github.com/gazebosim/gz-rendering/pull/940)

1. Add minor comments to BaseGizmoVisual
    * [Pull request #881](https://github.com/gazebosim/gz-rendering/pull/881)

1. Update github issue template
    * [Pull request #939](https://github.com/gazebosim/gz-rendering/pull/939)

1. Prepare for 6.6.2 release
    * [Pull request #930](https://github.com/gazebosim/gz-rendering/pull/930)

1. prepare for 6.6.2 release
    * [Pull request #prepare for 6.6.2 release](https://github.com/gazebosim/gz-rendering/pull/prepare for 6.6.2 release)
```

I'll send a PR for fixing the Changelog yes.

Reply:
I just created a PR for this. #960

Reply:
thanks Ian!